src/test/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionControllerTest.java

```diff
@@ -87,18 +90,33 @@
 
     @Test
     void createObjectionTest() {
+        ObjectionCreate objectionCreate = new ObjectionCreate();
+        objectionCreate.setFullName(FULL_NAME);
+        objectionCreate.setShareIdentity(false);
         Objection objection = new Objection();
         objection.setId(OBJECTION_ID);
         objection.setStatus(ObjectionStatus.SUBMITTED);
-        when(objectionService.createObjection(REQUEST_ID, COMPANY_NUMBER, AUTH_ID, AUTH_USER)).thenReturn(objection);
+
+        when(objectionService.createObjection(
+                eq(REQUEST_ID),
+                eq(COMPANY_NUMBER),
+                eq(AUTH_ID),
+                eq(AUTH_USER),
+                any()
```

Comment:
instead of any() you should use the objectionCreate that is created above that will be passed in as a param

Reply:
Updated.